eval.py

```diff
@@ -331,7 +331,7 @@ def evaluate():
     summary_writer = tf.summary.FileWriter(
         os.path.join(FLAGS.train_dir, "eval"), graph=tf.get_default_graph())
 
-    evl_metrics = eval_util.EvaluationMetrics(reader.num_classes, FLAGS.top_k)
+    evl_metrics = eval_util.EvaluationMetrics(reader.num_classes, FLAGS.top_k, None)
```

Comment:
Hi, 

Would you mind to reformat it (you can copy paste from below) to 

```python
    evl_metrics = eval_util.EvaluationMetrics(reader.num_classes, FLAGS.top_k,
                                              None)
```

due to 80-char length limit here? 

http://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
did it in a new pull request #91